main.py

```diff
@@ -45,7 +45,7 @@ def print_results(self, results):
 
 def main():
     crawler = WebCrawler()
-    start_url = "https://example.com"
+    start_url = "https://www.flipkart.com" #valid url is taken
```

Comment:
https://example.com is a valid url, you can check it in the browser, a webpage gets opened. Its not a bug!